app/controllers/api/v1/list_items_controller.rb

```diff
@@ -32,7 +32,12 @@ def create
     end
   end
 
+  def edit
+    @list_item = ListItem.find(params[:id])
+  end

```

Comment:
Why do you need the `edit` endpoint? Rails already has a `show` endpoint by default right?

`GET /list_item/:id`